MSI/Day1/문자열내p와y의개수.js

```diff
@@ -3,7 +3,7 @@ function solution(s) {
     let yCount = 0; 
 
     for (let char of s) {
```

Comment:
```suggestion
    for (const char of s) {
```
- 값이 변하지 않는 변수는 const를 1순위로 선언해주세요 :)

Reply:
넵 const 사용하겠습니다 감사합니다 !!